packages/components/src/components/button/button.jsx

```diff
@@ -30,8 +30,8 @@ const Button = ({
     type,
     primary,
     secondary,
-    small,
     tertiary,
+    small,
```

Comment:
Could you help to move it to move it back to the previous line? So that it will keep alphabetical order.